drivers/cc110x/cc110x_rx_tx.c

```diff
@@ -0,0 +1,326 @@
+/*
+ * Copyright (C) 2018 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup   drivers_cc110x
+ * @{
+ *
+ * @file
+ * @brief     Functions to manage sending/receiving frames with the CC110x
+ *
+ * @author    Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ * @}
+ */
+
+#include "xtimer.h"
+#include "cc110x.h"
+#include "cc110x_internal.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+/* Use NETDEV_EVENT_ISR to indicate that no event needs to be passed to upper
+ * layer at end of ISR, as ISR will never need this event
+ */
+#define NETDEV_NO_EVENT NETDEV_EVENT_ISR
+
+void cc110x_rx(cc110x_t *dev)
+{
+    //DEBUG("[cc110x] Going to RX\n");
```

Comment:
Please uncomment this line.

Reply:
Done in https://github.com/RIOT-OS/RIOT/pull/10340/commits/b840e71811a77d64d0de67a9706e96082276915c